<launch>
  <arg name="robot_name"   default="open_manipulator"/>
  <arg name="end_effector" default="gripper"/>

  <group ns="$(arg robot_name)">
    <node name="modified_teleop" pkg="modified_teleop" type="modified_teleop" output="screen">
      <remap from="kinematics_pose" to="$(arg end_effector)/kinematics_pose"/>
      <param name="end_effector_name" value="$(arg end_effector)"/>
    </node>
  </group>
</launch>
